### PR TITLE
[EASI-3675] Updating Risk Type Questions - FE

### PR DESCRIPTION
**Pull request by garyjzhao**
# EASI-3675

## Changes and Description

- Updated Risk Type question to match AC
- Updated Risk Type readonly view
- Removed participantAssuemRisk throughout app
- Updated snaps and tests

## How to test this change

1. Go to page 3 of Partipcant and Providers
2. Check to see question matches Figma
3. Check to see data persist, shows up on read only and CSV

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
Nice Gary

I'm going to request some changes pending on a different PR that will addressed some unwanted behavior with the ordering of the values for your work.

Here is the explanation of the issue and the work - [https://github.com/CMSgov/mint-app/pull/877](https://github.com/CMSgov/mint-app/pull/877)

Here is an example of the incorrect ordering

Screenshot 2023-12-29 at 10 12 58 AM

**Comment by patrickseguraoddball** [comment]
One other thing to note - I see you fixed the bullet not rendering on the readonly.  I like your style solution to this over the one I fixed in main - [https://github.com/CMSgov/mint-app/pull/870](https://github.com/CMSgov/mint-app/pull/870).  Then main get's merged into this, will most likely be a merge conflict.  Feel free to choose/merge your solution over mine!

**Comment by StevenWadeOddball** [comment]
@garyjzhao, can you update the feature branch with main, and then merge the feature branch into this branch? When you merge the main branch into this branch based off of a feature, it can result in incorrect GIT history.

File: src/utils/modelPlan.ts [hunk 1]
```diff
@@ -168,6 +168,13 @@ export const sortOtherEnum = (a: string, b: string) => {
...
 };
 
+export const sortOtherLast = (a: string, b: string) => {
+  if (b === 'OTHER') {
+    return -1;
+  }
+  return 0;
+};
+
 // Sort possible operational needs
```
Comment (patrickseguraoddball) [inline src/utils/modelPlan.ts:178]: We no longer need this util if we are using the sort from the translations

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]

**Review by patrickseguraoddball — APPROVED** [review]
LG!

**Pull request merged**